x-pack/auditbeat/module/system/process/_meta/data.json

```diff
@@ -0,0 +1,29 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
```

Comment:
`agent.type:auditbeat`, with `agent.name` defaulting to the same value, or absent. I'd go for absent in the case of Auditbeat. This pattern around `.name` needs to be better documented (see https://github.com/elastic/ecs/issues/269).

`agent.hostname` should instead be `host.hostname`

Reply:
Those are standard values that the testing framework puts in. The actual data has:
```
"agent": {
    "hostname": "5efeb8ba7b09",
    "version": "7.0.0",
    "type": "auditbeat"
  },
```

We should probably adjust the testing framework.

Reply:
Gotcha.

This snippet you pasted is what live Auditbeat generates, though? If that's the case, `agent.hostname` still needs to move to `host.hostname`.

The definition of `agent` is that it's a monitor that runs on the host, so there's no need for hostname there. This is in contrast to `observer`, which is defined as a separate host or device that monitors a system from the outside.

Reply:
Fair enough. This is in `master` today (introduced last month with https://github.com/elastic/beats/pull/8873) and affects all Beats - we should address it separately.